qiskit/primitives/base/base_estimator.py

```diff
@@ -239,8 +189,7 @@ def run(
             **run_opts.__dict__,
         )
 
-    # This will be comment out after 0.22. (This is necessary for the compatibility.)
-    # @abstractmethod
+    @abstractmethod
```

Comment:
I am not sure if you want to do anything about the raise in the body here (same in base_sampler) - python now will not let you instantiate any class without it implemented right so the only time you might see this is looking in this source code if some subclass you created was failing and you had not dealt with the deprecation yet. So if its more documenting that this was changed to abstract that did not (will not) happen until 0.24 right - not 0.22.

Reply:
I am late in removing my comment because of the delay in updating the other libraries. Well, 0.24 would also be after 0.22.